libs/ui/inputs/src/lib/star-toggle/star-toggle.component.html

```diff
@@ -5,12 +5,15 @@
     (click)="toggle($event)"
     [ngClass]="{
       enabled: toggled,
-      'disabled opacity-45': !toggled,
+      disabled: !toggled,
       'transition hover:scale-125 will-change-transform': !disabled,
       'cursor-default': disabled,
     }"
   >
-    <ng-icon [name]="toggled ? 'matStar' : 'matStarBorder'"></ng-icon>
+    <ng-icon
+      [name]="toggled ? 'matStar' : 'matStarBorder'"
+      size="24px"
+    ></ng-icon>
```

Comment:
isn't that going to change the appearance in the search results as well?

Reply:
It's actually the size they are already displayed in, in the search, but for some reason they shrank in the nav bar.

Reply:
maybe because the size of the star was proportional to the text size, which is really handy for UI components; fixing a size in px here means it's going to be impossible to have a star of a different size somewhere else

Reply:
Ok I'll see if I get it sized via the text size